build/lib/pgBackRestBuild/Config/Data.pm

```diff
@@ -0,0 +1,2234 @@
+    push @EXPORT, qw(CFGBLDOPT_REPO_PATH);
+use constant CFGBLDOPT_REPO_TYPE                                       => 'repo-type';
+    push @EXPORT, qw(CFGBLDOPT_REPO_TYPE);
+
+# Repository S3
+use constant CFGBLDOPT_REPO_S3_KEY                                     => 'repo-s3-key';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_KEY);
+use constant CFGBLDOPT_REPO_S3_KEY_SECRET                              => 'repo-s3-key-secret';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_KEY_SECRET);
+use constant CFGBLDOPT_REPO_S3_BUCKET                                  => 'repo-s3-bucket';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_BUCKET);
+use constant CFGBLDOPT_REPO_S3_CA_FILE                                 => 'repo-s3-ca-file';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_CA_FILE);
+use constant CFGBLDOPT_REPO_S3_CA_PATH                                 => 'repo-s3-ca-path';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_CA_PATH);
+use constant CFGBLDOPT_REPO_S3_ENDPOINT                                => 'repo-s3-endpoint';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_ENDPOINT);
+use constant CFGBLDOPT_REPO_S3_HOST                                    => 'repo-s3-host';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_HOST);
+use constant CFGBLDOPT_REPO_S3_REGION                                  => 'repo-s3-region';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_REGION);
+use constant CFGBLDOPT_REPO_S3_VERIFY_SSL                              => 'repo-s3-verify-ssl';
+    push @EXPORT, qw(CFGBLDOPT_REPO_S3_VERIFY_SSL);
+
+# Log level
+use constant CFGBLDOPT_LOG_LEVEL_CONSOLE                               => 'log-level-console';
+    push @EXPORT, qw(CFGBLDOPT_LOG_LEVEL_CONSOLE);
+use constant CFGBLDOPT_LOG_LEVEL_FILE                                  => 'log-level-file';
+    push @EXPORT, qw(CFGBLDOPT_LOG_LEVEL_FILE);
+use constant CFGBLDOPT_LOG_LEVEL_STDERR                                => 'log-level-stderr';
+    push @EXPORT, qw(CFGBLDOPT_LOG_LEVEL_STDERR);
+use constant CFGBLDOPT_LOG_TIMESTAMP                                   => 'log-timestamp';
+    push @EXPORT, qw(CFGBLDOPT_LOG_TIMESTAMP);
+
+# ARCHIVE Section
+#-----------------------------------------------------------------------------------------------------------------------------------
+use constant CFGBLDOPT_ARCHIVE_ASYNC                                   => 'archive-async';
+    push @EXPORT, qw(CFGBLDOPT_ARCHIVE_ASYNC);
+# Deprecated and to be removed
+use constant CFGBLDOPT_ARCHIVE_MAX_MB                                  => 'archive-max-mb';
+    push @EXPORT, qw(CFGBLDOPT_ARCHIVE_MAX_MB);
+use constant CFGBLDOPT_ARCHIVE_QUEUE_MAX                               => 'archive-queue-max';
+    push @EXPORT, qw(CFGBLDOPT_ARCHIVE_QUEUE_MAX);
+
+# BACKUP Section
+#-----------------------------------------------------------------------------------------------------------------------------------
+use constant CFGBLDOPT_ARCHIVE_CHECK                                => 'archive-check';
```

Comment:
There were a few options that deviated from the general naming convention and included the command name.  This did not play well with the auto-generate of constants so was fixed.  I think it was just this, archive-copy, and restore-option that were affected.

Like I said, I'm still renaming stuff in this module and plan to reformat when I'm done.